VAMobile/src/screens/BenefitsScreen/Letters/LettersListScreen.test.tsx

```diff
@@ -143,163 +179,113 @@ context('LettersListScreen', () => {
 
     props = mockNavProps()
 
-    component = render(<LettersListScreen {...props} />, {
+    render(<LettersListScreen {...props} />, {
       preloadedState: {
         ...storeVals,
       },
     })
-
-    testInstance = component.UNSAFE_root
   }
 
-  it('initializes correctly', async () => {
-    await waitFor(() => {
-      initializeTestInstance(lettersData)
-      expect(component).toBeTruthy()
+  describe('when lettersAndDocuments is set to false', () => {
+    it('should show noLettersScreen', async () => {
+      initializeTestInstance(lettersData, false, false)
+      expect(screen.getByText("We couldn't find information about your VA letters")).toBeTruthy()
     })
   })
 
   describe('when loading is set to true', () => {
     it('should show loading screen', async () => {
-      await waitFor(() => {
-        initializeTestInstance(lettersData, true)
-        expect(testInstance.findByType(LoadingComponent)).toBeTruthy()
-      })
+      initializeTestInstance(lettersData, true)
+      expect(screen.getByText('Loading your letters list...')).toBeTruthy()
     })
   })
 
   it('should display the correct list of letters', async () => {
     await waitFor(() => {
```

Comment:
actually all of those await waitFors are needed, the testing for all of the tests was a PAIN to get through and @theodur  could also account for that lol

Reply:
In this case, those actually might not be needed. Since it's just setting things up and rendering the component, we shouldn't need to wait for that since nothing async is happening, or at least I didn't need to for the tests I've worked on. I've had to wait for the `expect` calls in certain scenarios, though. I noticed those `await waitFor()` calls on `initializeTestInstance` were there prior to this PR. Does removing them cause failures for you @Sparowhawk?